wagtailio/services/templates/services/blocks/card.html

```diff
@@ -0,0 +1,16 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+{% with value as card %}
+<div class="card">
+    <div class="card__upper">
+        <div>
+            {% image card.image fill-138x110%}
+        </div>
+        <hr class="card__divider">
+        <div class="card__text">
+            {{ card.text }}
+        </div>
+    </div>
+    <a class="card__link" href="{{card.link}}"></a>
```

Comment:
suggestion: i think it would be nicer here to make `.card` the anchor element, so something like:
```
<a class="card" href="{{card.link}}">
    <div class="card__upper">
            {% image card.image fill-138x110%}
        <hr class="card__divider">
        <div class="card__text">
            {{ card.text }}
        </div>
    </div>
    <div class="card__button">{{ card.link_text }}
        <svg class="external-link-icon" viewBox="0 0 100 100"><use href="#external-link"></svg>
    </div>
</a>
```
some styles might need tweaking as a result